code/android-core-library/src/main/java/com/adobe/marketing/mobile/util/TimeUtils.kt

```diff
@@ -20,10 +20,13 @@ import java.util.TimeZone
 
 object TimeUtils {
     private const val MILLISECONDS_PER_SECOND = 1000L
-    private const val ISO8601_DATE_FORMATTER_TIMEZONE_RFC822 = "yyyy-MM-dd'T'HH:mm:ssZZZ"
-    private const val ISO8601_DATE_FORMATTER_TIMEZONE_ISO8601 = "yyyy-MM-dd'T'HH:mm:ssXXX"
-    private const val RFC2822_DATE_PATTERN = "EEE, dd MMM yyyy HH:mm:ss z"
-    private const val ISO8601_DATE_FORMATTER_TIMEZONE_UTC = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+
+    enum class DatePattern(val pattern: String) {
```

Comment:
I would prefer private string constants instead of enum. This will allow other extensions to use `getFormattedString` even with formats not declared here.